docs/src/components/InteractiveExample/index.tsx

```diff
@@ -15,6 +15,68 @@ import ResetDark from '@site/static/img/reset-dark.svg';
 
 import styles from './styles.module.css';
 
+import ts from "typescript";
+import prettier from "prettier/standalone";
+import babelParser from 'prettier/parser-babel';
+
+import { lightGreen } from '@mui/material/colors';
+import { plugins } from '@site/babel.config';
+
+function compileTSXtoJSX(tsxCode) {
+  const TEXT_TO_REPLACE_1 = "// 1-COMMENT-TO-REPLACE";
+  const TEXT_TO_REPLACE_2 = "// 2-COMMENT-TO-REPLACE";
+
+  tsxCode = tsxCode.split("\n").map((line) => line.trim() === "" ? TEXT_TO_REPLACE_1 : line + TEXT_TO_REPLACE_2).join("\n");
```

Comment:
This seems to be a bit confusing, can you explain why you add comments at the end of line?

Reply:
Comments are added before ts compilation to keep code structure in output (empty lines etc.)